docs/getting_started/quickstart.rst

```diff
@@ -72,7 +75,7 @@ be obtained. For more information, see :ref:`reference_files_crds`.
 **4. Running the Pipeline**
 
 With `jwst` installed and CRDS configured for JWST, you can now run the pipeline
-and use JWST `datamodels`.
+and use JWST `gatamodels`.
```

Comment:
`gatamodels`? guessing a bad keystroke here.

Reply:
Those are the datamodels for use in Florida.